config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to nothing/blank (i.e. elasticsearch.requestHeadersWhitelist: )
+# or the empty list (i.e. elasticsearch.requestHeadersWhitelist: [])
```

Comment:
Not particularly. I think as a user its intuitive for blank values to mean nothing (in addition to `[]` also meaning list with nothing in it) so I documented the alternative. I'm happy to only document the `[]` option (even though the blank would still work but might be less confusing to the user if we don't actually document it).